src/vrl/selectielijst/models.py

```diff
@@ -97,6 +98,10 @@ class Resultaat(models.Model):
     vhrosv = models.BooleanField(_("VHROSV"), default=False)
     heffen_belastingen = models.BooleanField(_("heffen belastingen etc."), default=False)
     alle_taakgebieden = models.BooleanField(_("alle taakgebieden"), default=False)
+    procestermijn_opmerking = models.CharField(
+        _("procestermijn opmerking"), max_length=20, null=True, blank=True,
```

Comment:
Text-based fields (CharField, TextField, EmailField etc.) should not have `null=True`, because then you end up with two possible values for 'empty': `None` and `''`.